app/src/main/java/com/example/androidapp/Task.java

```diff
@@ -29,6 +30,8 @@ public String getLocation() {
         return this.location;
     }
 
+    public boolean isCompleted(){return this.completed; }
```

Comment:
Stick to the code style. This should be escaped, as all the other methods are defined.